components/support/guid/src/lib.rs

```diff
@@ -0,0 +1,482 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
```suggestion

#![allow(unknown_lints)]
#![warn(rust_2018_idioms)]
```
to fit with our other crates.